lib/logger/utils.ts

```diff
@@ -62,9 +74,20 @@ function sanitizeValue(value: any, seen = new WeakMap()): any {
     const objectResult: Record<string, any> = {};
     seen.set(value, objectResult);
     for (const [key, val] of Object.entries<any>(value)) {
-      objectResult[key] = seen.has(val)
-        ? seen.get(val)
-        : sanitizeValue(val, seen);
+      let curValue: any;
+      if (redactedFields.includes(key)) {
+        curValue = '***********';
+      } else if (contentFields.includes(key)) {
+        curValue = '[content]';
+      } else if (templateFields.includes(key)) {
+        curValue = '[Template]';
+        // } else if (arrayFields.includes(key)) {
+        //   curValue = '[Array]';
```

Comment:
Left from development?

Reply:
Not sure if we want this here. Can be removed